tests/secrets/test_runner.py

```diff
@@ -63,7 +80,7 @@ def test_runner_multiple_files(self):
         self.assertEqual(4, len(report.failed_checks))
```

Comment:
@gruebel After merging from master, can you update this to 5 please?

Reply:
yeah sure, give me a few minutes 😄